intercoop_addons/pos_product_barcodes/__openerp__.py

```diff
@@ -13,7 +13,9 @@
     'website': 'http://www.lalouve.net',
     'depends': [
         'base',
-        'barcodes'
+        'barcodes', 
+        'stock',
+        'point_of_sale'
```

Comment:
Just for information, to get a nice git diff you can add the coma after ``'point_of_sale',`` so next time you add a dependency at the end of the list you only get the diff on the additional module. I know this is perfectionism!